web-components/src/components/combobox/ComboBox.test.ts

```diff
@@ -1543,26 +1541,7 @@ describe("Combobox Component", () => {
   });
 
   describe("Combobox with virtual scroll", () => {
-    test("should correct render options for virtual scroll", async () => {
```

Comment:
Dont we need this test case?

Reply:
No, we dont need it as we are doing the same thing in next test case